Upstream has moved to pyo3 0.25.0, which compiles with Python 3.14, but has not
tagged a release.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,7 +10,7 @@
 path = "src/rust/lib.rs"
 
 [dependencies]
-pyo3 = "0.22.5"
+pyo3 = "0.25.0"
 color-thief = "0.2"
 image = "0.25.4"
 itertools = "0.13.0"
